internal/database.go

```diff
@@ -80,7 +86,7 @@ func getSQLiteDialector() *gorm.Dialector {
 	return &sqlDialector
 }
 
-// getPostgresDialector creates a database connection and returns the gorm.Dialector instance
+// getPostgresDialector creates a gormDB connection and returns the gorm.Dialector instance
```

Comment:
Refactor Issue or intended?

Reply:
This one is intended. Variable `database` has been renamed to `gormDB` to prevent package name conflict.